src/library/scala/StringContext.scala

```diff
@@ -417,7 +415,7 @@ object StringContext {
     }
   }
 
-  private def standardInterpolator(process: String => String, args: scala.collection.Seq[Any], parts: Seq[String]): String = {
+  def standardInterpolator(process: String => String, args: scala.collection.Seq[Any], parts: Seq[String]): String = {
```

Comment:
Is this function made public just so the macros can call it?
If so, would it make sense to move it to `scala.runtime` and add scaladoc that it's not really meant for normal usage?

Reply:
Yes, but it's pretty benign. Other interpolators could potentially be built on top of it. I don't think we need to move this (which would require reSTARring again).

Reply:
ok. Then LGTM.